test_scripts/API/GetWayPoints/001_GetWayPoints_success_flow.lua

```diff
@@ -0,0 +1,90 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case:
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid and allowed by Policies GetWayPoints_request to SDL
+-- SDL must:
+-- 1) Transfer GetWayPoints_request to HMI
+-- 2) Respond with <resultCode> received from HMI to mobile application
+-- 3) Provide the requested parameters at the same order as received from HMI 
+--    to mobile application (in case of successfull response)
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_get_way_points = require('test_scripts/API/GetWayPoints/common_get_way_points')
```

Comment:
@LitvinenkoIra  I propose to use module name variable in camelCase:
`local commonWayPoints = require('test_scripts/API/GetWayPoints/common_get_way_points')` 
or
`local commonWP = require('test_scripts/API/GetWayPoints/common_get_way_points')`